plugin/src/main/java/com/denizenscript/denizen/npc/DenizenNPCHelper.java

```diff
@@ -164,9 +164,12 @@ public void onInventoryClick(InventoryClickEvent event) {
             NPCTag npc = (NPCTag) inventory.getHolder();
             npc.getInventory().setContents(inventory.getContents());
             Equipment equipment = npc.getEquipmentTrait();
-            for (int i = 0; i < 5; i++) {
```

Comment:
this whole section of code was apparently a 2014-era hackaround that shouldn't have still been present in Denizen. I removed it on dev, just remove the edits to this file in this PR to fix the merge